bot/exts/info/information.py

```diff
@@ -518,12 +518,12 @@ async def json(self, ctx: Context, message: Message) -> None:
         await self.send_raw_content(ctx, message, json=True)
 
     @command(aliases=("rule",))
-    async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
+    async def rules(self, ctx: Context, rules: Greedy[int], keyword: Optional[str]) -> None:
         """Provides a link to all rules or, if specified, displays specific rule(s)."""
```

Comment:
Once we finalise the desired interface for this command, please update the docstring to mention that keywords can be used. Keep in mind this docstring appears in the help information for the command.

Reply:
Noted, thanks !